src/TensorSharp/Core/MemoryCopier.cs

```diff
@@ -8,34 +8,9 @@ namespace TensorSharp.Core
 {
     internal static class MemoryCopier
     {
-        private static class NativeMethods32
+        public static unsafe void Copy(IntPtr destination, IntPtr source, ulong length)
         {
-            [DllImport("kernel32.dll")]
-            public static extern void CopyMemory(IntPtr destination, IntPtr source, uint length);
-        }
-
-        private static class NativeMethods64
-        {
-            [DllImport("kernel32")]
-            public static extern void RtlCopyMemory(IntPtr destination, IntPtr source, ulong length);
-        }
-
-        public static void Copy(IntPtr destination, IntPtr source, ulong length)
-        {
-            var is32 = IntPtr.Size == 4;
-
-            if (is32)
-            {
-                // Note: if this is run, length should always be in range of a uint
-                // (it should be impossible to allocate a buffer bigger than that range
-                // on a 32-bit system)
-                NativeMethods32.CopyMemory(destination, source, (uint)length);
-            }
-            else
-            {
-                NativeMethods64.RtlCopyMemory(destination, source, (ulong)length);
-            }
-
+            Buffer.MemoryCopy(source.ToPointer(), destination.ToPointer(), length, length);
```

Comment:
Since this is only called in two places (both in `src/TensorSharp/Cpu/CpuStorage.cs`), I think it would be cleaner to remove this method and inline the call to `Buffer.MemoryCopy` inside `CpuStorage.CopyToStorage` and `CpuStorage.CopyFromStorage`. Then this whole file can be removed too.

Reply:
removed